contractcourt/anchor_resolver.go

```diff
@@ -118,6 +118,13 @@ func (c *anchorResolver) Resolve() (ContractResolver, error) {
 			Fee: sweep.FeeEstimateInfo{
 				FeeRate: relayFeeRate,
 			},
+			// For normal anchor sweeping ,the budget is 330 sats.
```

Comment:
Nit: Can you be more specific what a normal anchor sweep is ?

Reply:
Deferred this to a new PR.